docs/source/tutorial/manage-access-control-list.txt

```diff
@@ -0,0 +1,187 @@
+===========================
+Manage Access to Data (ACL)
+===========================
```

Comment:
Rendered View:

https://kay.inrupt.net/public/docs-staging/solid-client-js/tutorial/manage-access-control-list.html

FYI - I'm using general terms for this draft instead of the more formal terms -- I first wanted to ensure that I'm understanding the concepts correctly. Once we're good, then I can quickly rewrite with the formal terminology.